settings/artix7_100t.sh

```diff
@@ -31,15 +31,4 @@ export XRAY_ROI_GRID_X2="77"
 export XRAY_ROI_GRID_Y1="0"
```

Comment:
Don't rename this file, or if you do, add another file like was done here: https://github.com/SymbiFlow/prjxray/blob/ba0ed9dba2f960b68e4d86e6963bbaf0c088466a/settings/spartan7.sh

Reply:
That's also the problem why the CI build failed. I thought it might be more intuitive that the 100t is not hidden in the artix7.sh. Will undo this, too.